pkg/nb/types.go

```diff
@@ -578,8 +578,19 @@ type UpdateBucketClassParams struct {
 
 // BucketReplicationParams is the params of bucket_api.put_bucket_replication()
 type BucketReplicationParams struct {
-	Name              string        `json:"name"`
-	ReplicationPolicy []interface{} `json:"replication_policy"`
+	Name                    string        `json:"name"`
+	ReplicationPolicy       []interface{} `json:"replication_policy"`
+	LogBasedReplicationInfo LogsInfo      `json:"logs_based_replication_info"`
+}
+
+// LogsInfo is a struct to define required information for log based replication
+type LogsInfo struct {
+
+	//bucket name of the bucket that holds the logs
+	LogsBucket string `json:"LogsBucket,omitempty"`
+
+	//subpath under the bucket to find the logs
+	SubPath string `json:"maxObjects,omitempty"`
```

Comment:
fix maxObjects to subpath also for both LogsBucket and SubPath it should be in lower case